core/gql_queries.py

```diff
@@ -33,7 +33,6 @@ class Meta:
         filter_fields = {
             "id": ["exact"],
             "name": ["exact", "istartswith", "icontains", "iexact"],
-            "is_system": ["exact"],
```

Comment:
why are you removing this filter, only update of "is system is forbidden"

Reply:
so as to not override the custom filter defined in schema.py in core module.